auditbeat/module/auditd/config_test.go

```diff
@@ -166,7 +166,7 @@ audit_rules: |
 			makeRuleFlags(0, 2),
 		}, "\n")
 
-		dir1, err := ioutil.TempDir("", "rules1")
+		dir1, err := os.MkdirTemp("", "rules1")
```

Comment:
It looks like it can here too, and L197. In the cases here, there is no clean up of the directories that are made, which is likely a bug that would be fixed by the replacement.